marklogic-data-hub-central/ui/src/pages/Modeling.tsx

```diff
@@ -168,56 +169,109 @@ const Modeling: React.FC = () => {
     Revert All
   </MLButton>;
 
+  const handleViewChange = (view) => {
```

Comment:
In handleViewChange, "graph" (or the config variable for it) should be the default so I would reverse the logic ("graph" as the fallback)?

Reply:
Done.